install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of an already running external Broker instance
                         type: string
                     type: object
                   native:
                     properties:
                       name:
+                        description: Name of the Broker instance to be deployed
```

Comment:
Is there a default name that is used, unless customized? Where does this name show up? Only in `kubectl get` or in Meshery UI, too?

Reply:
The Broker Instance name here is a required name(of the Broker instance) to be filled not a default name; The name shows up only in `kubectl get deployment -n *(namespace name)` not in the NATS connection of the Meshery UI.